README.md

```diff
@@ -58,7 +58,6 @@ You can click a link available in a netlify bot PR comment to see and review you
 4. `cd` into the `website` subdirectory: `cd website`
 5. Install the required node packages: `npm install` (optional &mdash; install any updates)
 6. Build the website: `npm start`
-7. Before pushing your changes to a branch, check that all links work by using the `make build` script.
```

Comment:
This line could also be updated to include both possible commands: `make build` or `npm run build`, rather than removing entirely. But if removing for any other reasons, disregard this comment!

Reply:
👍 updated that line and a couple others (to include the `make` commands) in 83e2610f27d61ba720330be050b144db2a602915